rebasebot/bot.py

```diff
@@ -147,16 +147,18 @@ def _do_rebase(gitwd, source, dest, source_repo, tag_policy):
     # This should be the list of commits we are carrying on top of the UPSTREAM
     commits = gitwd.git.log("--reverse", "--pretty=format:%H - %s", "--no-merges",
                             "--ancestry-path", f"{merge_base}..dest/{dest.branch}")
-    logging.info("Picking commits: \n%s", commits)
```

Comment:
Perhaps we can keep this but reword it somehow? I think it's useful to include a list of all the commits we have found that are to be carried?

Perhaps "Identified upstream commits" would work?